packages/frontend-near-wallet/src/modules/wallet-migration/components/MigrateAccounts.tsx

```diff
@@ -0,0 +1,116 @@
+
+import React from 'react';
+import Button from '../../../components/Button';
+import { styled } from '../../../styles';
+
+
+import IconAccount from '../../../assets/images/IconAccount';
+import IconMigrateAccount from '../../../assets/images/IconMigrateAccount';
+import { getAvailableAccounts } from '../../../utils/migration';
+
+const MigrateAccounts = ({ onContinue, onClose }) => {
+    const accounts = getAvailableAccounts()
+
+    return (
+        <StyledContainer>
+            <IconMigrateAccount />
+            <h3 className='ttl'>We found {accounts.length} active account(s)</h3>
+            <p className='desc'>
+                The following accounts will be transferred to MyNearWallet:
```

Comment:
i18n here? Speaking of which we should probably include a language select option on the new page. Maybe in a follow up? cc @agileurbanite @heycorwin @nall-near

Reply:
Agreed @esaminu, would be good to make that a standard practice across this flow eventually.

Reply:
To be handled in another task.

Reply:
Created a new issue to discuss and implement this #2887 @esaminu @gutsyphilip